libsplinter/src/admin/store/diesel/operations/upgrade.rs

```diff
@@ -67,6 +67,7 @@ impl<'a> AdminServiceStoreUpgradeProposalToCircuitOperation
                 .with_durability(proposed_circuit.durability())
                 .with_routes(proposed_circuit.routes())
                 .with_circuit_management_type(proposed_circuit.circuit_management_type())
+                .with_circuit_version(proposed_circuit.circuit_version())
```

Comment:
Shouldn’t the status be a outcome of the proposal’s application?

Reply:
The `status` is being set based on the `CircuitProposal` produced by the admin service, so if it is a `CircuitDisbandRequest`, the status of the `CircuitProposal` is `Disbanded`. Then, member nodes are able to vote on that proposal to actually disband the circuit, in which case the active `Circuit` in the admin store is updated to show the `disbanded` status.

Reply:
This fix (adding the `circuit_version` here) was a bug fix where the circuit version was not being accurately carried over when upgrading a proposal to a circuit. As the default of this column is 1, all circuits upgraded would default to 1 (despite possibly being a version 2 circuit) so this line fixes that, specifically.